plugins/cue_points/quiz/QuizPlugin.php

```diff
@@ -482,6 +485,12 @@ protected function getReportCount($objectIds)
 		$c = new Criteria();
 		$c->add(CuePointPeer::ENTRY_ID, $objectIds);
 		$c->add(CuePointPeer::TYPE, QuizPlugin::getCoreValue('CuePointType', QuizCuePointType::QUIZ_QUESTION));
+		$anonKuser = kuserPeer::getKuserByPartnerAndUid($dbEntry->getPartnerId(), 0, true);
```

Comment:
Change code 'kuserPeer::getKuserByPartnerAndUid($dbEntry->getPartnerId(), 0, true);' to getAnonimusKuser(partnerId);